### DISCUSSION TITLE
SugarSkull blown away

### DISCUSSION THREAD

**Discussion by omichel** [cyberbotics/wrestling#42] [General]
During its last game, SugarSkull literally exploded and actually won the game because its head was sent very far away, above the threshold.

Explosions are usually due to hard physics constraints that the physics engine of Webots cannot handle properly, thus causing the rigid body simulation model to blow away. It may be caused by some part of the robot being stuck below some stairs or the like while trying to move. Unfortunately, it is very difficult to prevent such explosions to happen. However, usually an exploded robot is often in a desperate situation where he could hardly recover.

Therefore, we will slightly modify the rule to detect exploded robots and consider them as KO.
This will prevent users to try to make their robot explode to win a game.

**Comment by rua0ra1** [comment]
self sacrifice for a greater outcome :P

**Comment by omichel** [comment]
This was fixed in #43. Let us know here if that change is causing any problem.
